.github/workflows/build-deploy-staging.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         id: tag
         uses: anothrNick/github-tag-action@1.13.0
         env:
-          RELEASE_BRANCH: 'staging'
+          RELEASE_BRANCH: staging
```

Comment:
seems unrelated - intentional?

Reply:
it was intentional, it is unrelated, but I wanted to get this fix in and figure it was small enough to not need to bother with a separate PR.

Reply:
👍